include/boost/url/url.hpp

```diff
@@ -1292,7 +1293,27 @@ class BOOST_SYMBOL_VISIBLE url
     //
     //--------------------------------------------
 
-    /** Normalize everything.
+    /** Normalize the URL components
+
+        Applies Syntax-based normalization to the
+        URL.
+
+        @li The scheme is normalized to lowercase
+        @li The host is normalized to lowercase
+        @li Percent-encoding triplets are normalized
+        to uppercase letters
+        @li Percent-encoded octets that correspond
+        to unreserved characters are decoded
+        @li Redundant path-segments are removed
```

Comment:
See, this documentation would be much nicer and clearer if it just said that it calls the functions `normalize_scheme`, `normalize_authority`, etc..  and then each of those functions could have more detailed docs.

Reply:
Yes. I'll make `normalize_scheme`, ... exist and then update that.